Singleton/SceneInstance.cs

```diff
@@ -2,16 +2,16 @@
 
 public class SceneInstance<T> : MonoBehaviour where T : class, new() {
 	//
-	private static T _Instance = null;
+	protected static T _Instance = null;
```

Comment:
Wouldn't you simply have a specific T for your child class?
In any case, where are you using it then? I don't see it being done in this PR.